src/Files.App/Helpers/ContextFlyoutItemHelper.cs

```diff
@@ -1105,6 +1105,13 @@ public static List<ContextMenuFlyoutItemViewModel> GetNewItemItems(BaseLayoutCom
 					ShowInZipPage = true,
 					IsEnabled = canCreateFileInPage
 				},
+				new ContextMenuFlyoutItemViewModel
+				{
+					Text = "BaseLayoutItemContextFlyoutShortcut/Text".GetLocalizedResource(),
```

Comment:
I'd change this to "Shortcut".GetLocalizedResource() so it's coherent with "Folder" and "File"